test/e2e_node/runner/remote/run_remote.go

```diff
@@ -623,24 +623,48 @@ func createInstance(imageConfig *internalGCEImage) (string, error) {
 	}
 	i.Scheduling = &scheduling
 	i.Metadata = imageConfig.metadata
+	var insertionOperationName string
 	if _, err := computeService.Instances.Get(*project, *zone, i.Name).Do(); err != nil {
 		op, err := computeService.Instances.Insert(*project, *zone, i).Do()
+
 		if err != nil {
 			ret := fmt.Sprintf("could not create instance %s: API error: %v", name, err)
 			if op != nil {
 				ret = fmt.Sprintf("%s: %v", ret, op.Error)
 			}
 			return "", fmt.Errorf(ret)
 		} else if op.Error != nil {
-			return "", fmt.Errorf("could not create instance %s: %+v", name, op.Error)
+			var errs []string
```

Comment:
Yes! Since `op.Error` is a a struct, and `op.Error.Errors` is a struct with pointers to structs, a normal `fmt.Printf` will only return the slice as a list of the pointer values.

So this will end up as: `[&{Code:QUOTA_EXCEEDED Location: Message:Quota 'CPUS' exceeded. Limit: 24.0 in region europe-west6. ForceSendFields:[] NullFields:[]}]`